ADOPTERS.md

```diff
@@ -89,6 +91,7 @@ If you would like to add your logo to a future `Adopters of Velero` section on [
 [50]: https://redhat.com
 [51]: https://github.com/fusor/mig-operator
 [52]: https://github.com/fusor/mig-operator/blob/master/docs/usage/2.md
+[53]: https://github.com/openshift/oadp-operator
```

Comment:
Should we update the fusor URL for MTC while we're at it?

Reply:
the fusor URL still redirects to konveyor :)